app/src/sharedTest/java/org/oppia/android/app/testing/NavigationDrawerTestActivityTest.kt

```diff
@@ -374,6 +374,23 @@ class NavigationDrawerTestActivityTest {
     }
   }
 
+  @Test
+  @Ignore("Navigation Drawer is not closing")
+  fun testNavigationDrawerTestActivity_selectSwitchProfileMenu_clickCancel_checkDrawerIsClosed() {
+    launch(NavigationDrawerTestActivity::class.java).use {
+      it.openNavigationDrawer()
+      onView(withText(R.string.menu_switch_profile)).perform(click())
+      onView(withText(R.string.home_activity_back_dialog_message))
+        .inRoot(isDialog())
+        .check(matches(isDisplayed()))
+      onView(withText(R.string.home_activity_back_dialog_cancel))
+        .inRoot(isDialog())
+        .perform(click())
+      testCoroutineDispatchers.runCurrent()
+      onView(withId(R.id.home_activity_drawer_layout)).check(matches(isClosed()))
```

Comment:
Out of curiosity, does this fail when you use it in the beginning of the test before the drawer is even opened?

Reply:
No, Initially this test case is never ignored and it always passes as the opening and closing has no effect and as result, the `home_activity_drawer_layout` is always in the closed state.

Reply:
That's what I was wondering. Keeping this as Espresso-specific to verify it fails correctly sgtm.